Source/SIMPLib/Geometry/ImageGeom.h

```diff
@@ -321,6 +321,11 @@ class SIMPLib_EXPORT ImageGeom : public IGeometryGrid
   void getCoords(size_t x, size_t y, size_t z, double coords[3]) const override;
   void getCoords(size_t idx, double coords[3]) const override;
 
+  virtual std::optional<size_t> getIndex(float coords[3]) const override;
+  virtual std::optional<size_t> getIndex(float xCoord, float yCoord, float zCoord) const override;
+  virtual std::optional<size_t> getIndex(double coords[3]) const override;
+  virtual std::optional<size_t> getIndex(double xCoord, double yCoord, double zCoord) const override;
```

Comment:
Methods don't need to be simultaneously virtual and override.

```suggestion
  std::optional<size_t> getIndex(float coords[3]) const override;
  std::optional<size_t> getIndex(float xCoord, float yCoord, float zCoord) const override;
  std::optional<size_t> getIndex(double coords[3]) const override;
  std::optional<size_t> getIndex(double xCoord, double yCoord, double zCoord) const override;
```